p4-16/spec/grammar.mdk

```diff
@@ -881,14 +881,14 @@ actionRef
 
 // BEGIN:entry
 entry
-    : optCONST optEntryPriority keysetExpression ':' actionRef optAnnotations ';'
+    : optCONST entryPriority keysetExpression ':' actionRef optAnnotations ';'
+    | optCONST keysetExpression ':' actionRef optAnnotations ';'
     ;
 // END:entry
 
 // BEGIN:optEntryPriority
```

Comment:
nit: Here and below, please replace optEntryPriority with entryPriority.

Reply:
Another good catch.  Definitely good for consistency's sake to fix that, and should be done with latest commit.

Reply:
Fixed by commit 2 on this PR.